python/graphstorm/model/lm_embed.py

```diff
@@ -388,9 +404,12 @@ def update_cache(self, lm_infer_batch_size, use_fp16=True):
                 logging.info('Computing bert embedding on node %s takes %.3f seconds',
                              ntype, time.time() - start)
             lm_model.train()
+        # store/update the lm model hash used for the cache
+        self._lm_hash = self._lm_models.get_all_lm_hashes()
```

Comment:
Is this the cause of the bug?

Reply:
not really. This statement runs when we need to compute BERT embeddings from scratch. We need to save the hash number of the current BERT models.

What we need to make sure is that when the BERT embeddings are loaded from files, we need to assign the right hash number for the loaded embeddings.